tests/integration/targets/flightctl_approve/tasks/certificate-signing-approval.yml

```diff
@@ -1,11 +1,12 @@
 ---
 - name: Test approve certificate signing request
   vars:
-    csr_name: ansible-integration-test-csr
+    csr_name: ansible-integration-test-approval-csr
   block:
   - name: Create a csr
     flightctl.edge.flightctl:
       flightctl_host: "{{ flightctl_host }}"
+      flightctl_validate_certs: false
```

Comment:
Skipping certs validation can be removed in the future when a plan for auth/certs is fully supported.